content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -261,7 +261,7 @@ model User {
 
 <Admonition type="warning">
 
-The `Restrict` action is **not** available on [SQL Server](../../../../database-connectors/microsoft-sql-server) and will trigger a schema validation error. Instead, you can use [`NoAction`](#noaction), which produces the same result and is compatible with SQLServer.
+The `Restrict` action is **not** available on [Microsoft SQL Server](../../../../database-connectors/microsoft-sql-server) and will trigger a schema validation error. Instead, you can use [`NoAction`](#noaction), which produces the same result and is compatible with SQLServer.
```

Comment:
Are we normalizing on "Microsoft SQL Server" everywhere?

Reply:
Yes